python/pyspark/sql/session.py

```diff
@@ -276,8 +315,20 @@ def getOrCreate(self) -> "SparkSession":
                     ).applyModifiableSettings(session._jsparkSession, self._options)
                 return session
 
-    builder = Builder()
-    """A class attribute having a :class:`Builder` to construct :class:`SparkSession` instances."""
+    # TODO: This workaround can be dropped once support for Python 3.8 is dropped
+    # In Python 3.9, the @property decorator has been made compatible with the @classmethod decorator.
+    # https://docs.python.org/3.9/library/functions.html#classmethod
+    if sys.version >= "3.9":
```

Comment:
Let's use this way: `sys.version_info >= (3, 9)`

Reply:
Sure, I'll do that.